ooniprobe/View/TestResults/Details/TestDetailsViewController.m

```diff
@@ -49,6 +50,13 @@ - (void)advancedScreens{
                                    actionWithTitle:NSLocalizedString(@"TestResults.Details.RawData", nil)
                                    style:UIAlertActionStyleDefault
                                    handler:^(UIAlertAction * action) {
+                                       if (![[ReachabilityManager sharedManager] isInternetAccessible]
```

Comment:
can you change the helper semantics and call it `noInternetAccess` so we avoid using negations?

Reply:
Should I add another function noInternetAccess or remove isInternetAccessible ?

Reply:
Let's just add what we need